pkg/virt-launcher/virtwrap/agent-poller/agent_parser.go

```diff
@@ -0,0 +1,153 @@
+package agentpoller
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+// GuestOsInfo is the response from 'guest-get-osinfo'
+type GuestOsInfo struct {
```

Comment:
Please keep this PR limited to change from simple to parallel poller. Are these new structures essential for the PR?

Reply:
It is only moved from `agent_poller.go`. Not introduced.